app/views/orders/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1> Buyer Details for Order #<%= @order.id %> </h1>
+
+<p>Name: <%= @order.name %></p>
+<p>Email: <%= @order.email %></p>
+<p>Address: <%= @order.address %></p>
+<p>CC: <%= @order.cc4.to_s.slice(-4..-1).to_i %></p>
```

Comment:
I think it would make sense to have a "short CC#" as a model property that would do this slicing logic for you